platform/ui/tailwind.config.js

```diff
@@ -665,9 +665,9 @@ module.exports = {
     backgroundSize: ['responsive'],
     borderCollapse: ['responsive'],
     borderColor: ['responsive', 'hover', 'focus', 'active'],
-    borderRadius: ['responsive', 'focus'],
+    borderRadius: ['responsive', 'focus', 'first', 'last'],
     borderStyle: ['responsive', 'focus'],
-    borderWidth: ['responsive', 'focus'],
+    borderWidth: ['responsive', 'focus', 'first', 'last'],
```

Comment:
Better to add these variants, or set them when iterating elements and hitting first/last in list?

Reply:
I followed what Rodrigo showed me when he showcased his tasks, I believe this variables increase the css file created for tailwind, so I would say, to create once we need.